Use explicitly the word Kubernetes in the migration guide. (#6380)
containous/traefik: docs/content/migration/v2.md

## v2.0 to v2.1

### Kubernetes CRD

In v2.1, a new Kubernetes CRD called `TraefikService` was added.
While updating an installation to v2.1,

> it is required to apply that CRD before as well as enhance the existing `ClusterRole` definition to allow Traefik to use that CRD.

Reviewer: ```suggestion
one should apply that CRD, and update the existing `ClusterRole` definition to allow Traefik to use that CRD.
```
